.github/workflows/release.yml

```diff
@@ -48,8 +48,15 @@ jobs:
         run: git restore .
 
       - run: npm run build
-      - run: npm run test
 
+      - name: Save full commit SHA
+        id: vars
+        run: echo ::set-output name=commit_sha::$(git rev-parse HEAD)
+        shell: bash
+      - name: Save commit in manifest file
+        run: echo '{"commit":"${{ steps.vars.outputs.commit_sha }}"}' > manifest.json
```

Comment:
Maybe we can use [this](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context:~:text=github.com.-,github.sha,-string) (`github.sha` from the github context) and skip the step of saving the SHA hash to a variable

Reply:
good point, updated.